17_differential_abundance.Rmd

```diff
@@ -39,7 +430,9 @@ tse <- SprockettTHData()
 names(colData(tse))
```

Comment:
I would hide this - secondary and takes lot of space.

Reply:
Same for line 441 and other similar lines. In most cases we do not need to show the assay. Let us try to highlight the most relevant content to understand the main features of the method, and hide other material in the background